packages/airnode-protocol/deployments/references.json

```diff
@@ -107,21 +134,30 @@
     "100": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
     "137": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
     "250": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
+    "288": "0x1d4F592E0723e03bed2Ff6d78F3CEe6750f08B38",
+    "416": "0xE338f63170c42bA0d2a888f18F6185369779009c",
     "420": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
     "588": "0x4923968942E8aae4656ae4913874EB7312e0F7c7",
     "1088": "0xC02Ea0f403d5f3D45a4F1d0d817e7A2601346c9E",
     "1284": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
     "1285": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
     "1287": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
+    "1294": "0xb3070A0F2f84765Ee19EfADf91dfE50690a9eEa1",
     "2001": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
     "4002": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
     "42161": "0xb015ACeEdD478fc497A798Ab45fcED8BdEd08924",
+    "42170": "0xd864A45334C7a632cA9149993682354D7f967F28",
     "43113": "0x7f5AF7a37a33898544717AAa6c35c111dCe95b28",
     "43114": "0xC02Ea0f403d5f3D45a4F1d0d817e7A2601346c9E",
+    "43288": "0xd864A45334C7a632cA9149993682354D7f967F28",
```

Comment:
How come this is not deterministically deployed while the other two contracts are? Now I can't run deploy:verify-local because it fails with the deterministic deployments and I can't check this by address because it's not deterministically deployed.

Reply:
this is deterministically deployed. If you scroll down [this](https://blockexplorer.avax.boba.network/address/0xBba33C9cB97a3eB9be85C5B9A8Da43D266d01c33) and see the last 3 contract call transactions those are deterministic deployments. I'm not exactly sure why the address is different but I've tested QRNG on all of the deployed chains and it works. Also `arbitrum-nova` is also deterministically deployed but the RRP address on that is the same as `Boba Avalanche`